src/mocks/syfomotebehov/historikkmotebehovMock.ts

```diff
@@ -1,27 +1,27 @@
 export const historikkmotebehovMock = [
   {
     opprettetAv: null,
-    tekst: "Møtebehovet ble lest av Z990000",
```

Comment:
Dette er ikke gyldige tekster som returneres fra esyfo, så byttet de for å unngå forvirring. Skal fjernes i neste PR.